media/windows/ricoh-printer/ricoh_db_create.py

```diff
@@ -0,0 +1,121 @@
+#!/user/bin/env python3
+
+# Script which runs daily to update staff print totals
+# based on CSVs generated by the printer itself.
+
+import sys
+sys.path.insert(1, '../../python/')
+
+import os
+import re
+import csv
+import datetime
+import sqlite3
+from sqlite3 import Error
+
+#import ECC
+#import PDSChurch
+
+from pprint import pprint
+from pprint import pformat
+
+###########################################################
+
+ricoh_db = "media/windows/ricoh-printer/ricoh.db"
+today = datetime.date.today()
+datestring = f'{today.year}{today.month}{today.day}'
+debug_datestring = '20201013'
+
+def _strip(value):
+    #The CSV puts brackets around values, which we don't want
+    return value.strip('[').strip(']')    
+
+def load_latest_csv():
+    
+    filename = f'RICOH IM C4500_usercounter_{datestring}.csv'
```

Comment:
The input CSV filename should probably be an input parameter to this function (because at some point, we'll have a different mechanism to download that CSV from the Ricoh).